### ISSUE TITLE
Support for several gitlab instances

### ISSUE THREAD

**Issue by jlengrand** [RamonGebben/mergify#10]
I actually use 3 different gitlab instances : 

* .com for open source / personal work
* local instance at my day job
* local instance at another job

It would be great to be able to select profiles, or at least use several instances at once in Mergify. 

Right now I can only see the merge requests for one instance :).

**Comment by RamonGebben** [comment]
Great suggestion, thanks!

**Comment by jlengrand** [comment]
Yeah,  I'm not straight offering help because I don't want to disappoint but I'll try to look into it. 

Just thinking quickly you could 
* Simply allow for several tokens and gitlab instances and show all
* Have some kind of profile flag `mergify --work`
* Something else? 

I haven't looked at the code yet so I don't know how I'd solve this right now :).

**Comment by RamonGebben** [comment]
Do you mix these "profiles" or would a `--set-profile work` also do the trick for you?

I think it would be very annoying to have to specify the profile for each command.

**Comment by jlengrand** [comment]
I think the easiest way to go first is to simply allow for several profiles and show them all. 

If I understand, what you propose is to set the profile and have one active at any time? 
Good idea too. I actually could use that because that would avoid having the command fail if I'm home without access to the work Gitlab.
